File: MangaViewer/MangaViewer/ViewModel/AppViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MangaViewer.ViewModel
{
    public class AppViewModel
    {
        #region MainViewModel
        private MainViewModel mainViewModel;
        public MainViewModel Main
        {
            get { return mainViewModel ?? (mainViewModel = new MainViewModel()); }
        }

        private SettingViewModel settingViewModel;
        public SettingViewModel Setting
        {
            get { return settingViewModel ?? (settingViewModel = new SettingViewModel()); }
        }
        #endregion

    }
}

File: MangaViewerAndroid/MangaViewer/src/com/king/mangaviewer/viewmodel/AppViewModel.java
package com.king.mangaviewer.viewmodel;

public class AppViewModel extends ViewModelBase {
	public MangaViewModel Manga;
	public SettingViewModel Setting;
	public LocalMangaViewModel LoacalManga;

	public AppViewModel() {
		this.Manga = new MangaViewModel();
		this.Setting = new SettingViewModel();
		this.LoacalManga = new LocalMangaViewModel();
	}
	
	
}
